charmcraft.yaml

```diff
@@ -150,3 +150,13 @@ requires:
     optional: true
     description: |
       Integration to help users discover Mimir's deployment.
+config:
+  options:
+    max_global_exemplars_per_user:
+      default: 0
+      description: |
+       The global limit for the number of exemplars.
+       When unset or set to a non-positive number, exemplar storage is disabled.
+       If set to a positive number that falls between 1 and 100,000, the limit is bumped to 100,000.
+       To disable, set to a non-positive number.
```

Comment:
I would remove this line: to disable, they should either set to 0 or unset; you can probably remove this entirely.